cursorless-talon/src/get_grapheme_spoken_form_entries.py

```diff
@@ -63,7 +68,7 @@ def get_id_to_spoken_form_map(list_name: str) -> Mapping[str, list[str]]:
     list to the list of spoken forms that map to the given value.
     """
     try:
-        raw_list = typing.cast(dict[str, str], registry.lists[list_name][0]).copy()
+        raw_list = typing.cast(dict[str, str], registry.lists[list_name][-1]).copy()
```

Comment:
```suggestion
        # NB: [-1] because the last list is the active one
        raw_list = typing.cast(dict[str, str], registry.lists[list_name][-1]).copy()
```